src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.icon {
+  width: 19px;
+  position: absolute;
+  top: 24px;
+  left: 25px;
+}
+
+.searchbar {
+  position: absolute;
+  top: 22px;
+  left: 62px;
+  border: none;
+  outline: none;
+  background: transparent;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchbar::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+
+}
+
+.container:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.container:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.container--small {
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
don't duplicate styles for `container--small` which you already have in `container`. check in other places